src/main/java/base/navigationservice/KeyboardListener.java

```diff
@@ -24,7 +24,7 @@ public void keyPressed(KeyEvent event) {
         if (keyCode < keys.length) {
             keys[keyCode] = true;
         }
-        if (keys[KeyEvent.VK_CONTROL] && keys[KeyEvent.VK_S]) {
+        if (keys[KeyEvent.VK_F5]) {
             saving = true;
```

Comment:
Looks good. You even changed tips, nice.

Can you also remove this "saving" variable completely? It is not needed anymore :)